### PR TITLE
Create .htaccess

### PR DESCRIPTION
**Pull request by irnerio-mosaico-opendata**
Add redirect for irnerio-mosaico ontology mmdio

### CODE REVIEW

**Comment by irnerio-mosaico-opendata** [comment]
@perma-id/w3id.org-maintainers Could you please review this pull request urgently? I need the link for my paper submission process. Thank you!

**Comment by dgarijo** [comment]
@irnerio-mosaico-opendata have you checked this htaccess file? It does not look like the redirection is ok. It should be:
```
RewriteRule ^$ https://github.com/irnerio-mosaico-opendata/mmdio/raw/main/docs/current/mmdio.html [R=302,L]
```
And that takes you to a raw html file. You should publish it with github pages and then you'll be able to see it properly

**Comment by dgarijo** [comment]
Please also add your user id in the contact details

File: irnerio-mosaico/ontology/mmdio/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Permanent Identifier for irnerio-mosaico ontology mmdio
+
+This identifier redirects to the mmdio ontology documentation for irnerio-mosaico.
+
+## Maintainer
+- [Your Name](faria.ferooz2@unibo.it)
+- [Your Organization](https://www.unibo.it/en)
```
Comment (TallTed) [inline irnerio-mosaico/ontology/mmdio/README.md:7]:
```suggestion
- [Faria Ferooz](faria.ferooz2@unibo.it)
- [Università di Bologna](https://www.unibo.it/en)
```

Comment (irnerio-mosaico-opendata) [reply irnerio-mosaico/ontology/mmdio/README.md]: @dgarijo I have updated the pull request to include my user ID and contact details in the README.md file

File: irnerio-mosaico/ontology/mmdio/README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Permanent Identifier for irnerio-mosaico ontology mmdio
+
+This identifier redirects to the mmdio ontology documentation for irnerio-mosaico.
+
+## Maintainer
+- # User ID: irnerio-mosaico-opendata(https://github.com/irnerio-mosaico-opendata)
+- #Email: faria.ferooz2@unibo.it
+- #Organization: https://www.unibo.it/en)
```
Comment (TallTed) [inline irnerio-mosaico/ontology/mmdio/README.md:8]:
```suggestion
- GitHub User ID: @irnerio-mosaico-opendata
  - Email: faria.ferooz2@unibo.it
  - Organization: https://www.unibo.it/en)
```

File: irnerio-mosaico/ontology/mmdio/.htaccess [hunk 3]
```diff
@@ -0,0 +1 @@
+RewriteRule ^$ https://github.com/irnerio-mosaico-opendata/mmdio/raw/main/docs/current/mmdio.html [R=302,L]
```
Comment (davidlehn) [inline irnerio-mosaico/ontology/mmdio/.htaccess:1]: Looks like you do have pages enabled.  As @dgarijo mentioned, this link is to the raw html.  You probably want this:
```suggestion
RewriteRule ^$ https://irnerio-mosaico-opendata.github.io/mmdio/docs/current/mmdio.html [R=302,L]
```

Comment (irnerio-mosaico-opendata) [reply irnerio-mosaico/ontology/mmdio/.htaccess]: Thank you so much!! It's working correctly

**Pull request merged**